### ISSUE TITLE
Question about pretrained DispNet's accuracy and training strategy

### ISSUE THREAD

**Issue by sunshinnnn** [CVLAB-Unibo/Real-time-self-adaptive-deep-stereo#25]
Hi, I wanna ask a question about DispNet. Have you test the KITTI2015 pretrained model, have it achieved the performance illustrated in the original paper in 2016CVPR, and what's your **training strategy** during the both syn and kitti period. Thank you!

**Comment by AlessioTonioni** [comment]
Hi,
yes we benchmarked our trained Dispnet model on the KITTI test set and we obtained results slightly worse than the original Dispnet implemented in caffe. The main difference between our implementation and the original Dispnet (besides the framework) is a different set of data augmentation operations performed at training time.

As for the training strategy we followed the one detailed in the original Dispnet paper.

**Comment by sunshinnnn** [comment]
yeah, THANKS for your reply. I could only find the strategy for training it on only the syn datas in the paper(write down there). 

However, I couldn't find the finetune strategy, maybe because of my carelessness, could you figure me out for that or if that does not exist in the paper, what is your own strategy(for example, followed training strategy in the paper on the pretrained period until model converged, and finetuned with a learning rate at ... for ... epoch,,,,,,,,just a example, not the pratical strategy)
THANK YOU!

**Comment by AlessioTonioni** [comment]
We have used the same strategy used for MADNet, so 500K iterations optimizing a loss function computed only on the full resolution prediction (i.e. the last one)

More details in the supplementary material of the paper.
